resources/assets/sass/components/_search.scss

```diff
@@ -0,0 +1,407 @@
+    		height: 18px;
+
+			@include respond-to(tablet) {
+
+				margin: 8px;
+	        	width: 32px;
+	        	height: 32px;
+			}
+
+        	.c-search.m-small & {
+
+				margin: 3px;
+        		width: 18px;
+        		height: 18px;
+        	}
+        }
+
+        .c-search.m-active.m-red & {
+
+            svg {
+
+            	fill: $brand-red;
+            }
+        }
+
+        .c-search.m-active.m-blue & {
+
+            svg {
+
+            	fill: $brand-blue;
+            }
+        }
+
+        .c-search.m-active.m-green & {
+
+            svg {
+
+            	fill: $brand-green;
+            }
+        }
+
+        .c-search.m-active.m-orange & {
+
+            svg {
+
+            	fill: $brand-orange;
+            }
+        }
+
+        .c-search.m-active.m-purple & {
+
+            svg {
+
+            	fill: $brand-purple;
+            }
+        }
+
+        .c-search.m-active.m-yellow & {
+
+            svg {
+
+            	fill: $brand-yellow;
+            }
+        }
+    }
+
+	.c-search__form-input {
+
+		color: $color-white;
+		font-size: $font-size-normal;
+		font-weight: $font-weight-medium;
+		width: 100%;
+		padding: 12px 12px 9px 48px;
+		background: rgba($color-white, 0.2);
+		border: none;
+		box-shadow: 0 18px 48px rgba($shadow-dark, 0.2);
+
+		@include respond-to(tablet) {
+
+			font-size: $font-size-larger;
+			padding: 18px 24px 12px 88px;
+		}
+
+		.c-search.m-small & {
+
+			font-size: $font-size-normal;
+			padding: 9px 12px 6px 48px;
+			background: none;
+			box-shadow: none;
+		}
+
+		&::input-placeholder {
+
+			color: $color-white;
+		}
+
+        .c-search.m-active & {
+
+            color: $color-text-black;
+            background: $color-white;
+
+            &::input-placeholder {
+
+                color: $color-text-neutral;
+            }
+            
+        }
+	}
+
+// Results
+
+.c-search__results {
+
+	position: absolute;
+	z-index: 1000;
+	display: none;
+	width: 100%;
+	background: $color-fill-neutral-light;
+	border-top: 1px solid $color-fill-neutral;
+	box-shadow: 0 18px 48px rgba($shadow-dark, 0.2);
+
+	.c-search.m-active & {
+
+		display: block;
+	}
+}
+
+	.c-search__results-content {
+
+		overflow: scroll;
+		height: 384px;
+		padding-bottom: 24px;
+	}
+
+		.c-search__results-list {
+
+			@include clear-list;
+
+			position: relative;
+			margin: 12px 0 0 0;
+			padding: 12px 24px 6px 88px;
+			border-bottom: 1px solid $color-fill-neutral;
+
+			.c-search.m-small & {
+
+				margin: 6px 0 0 0;
+				padding: 6px 12px 0 48px;
+			}
+
+			&:last-child {
+
+				border: none;
+			}
+
+			&:before {
+
+				content: '';
+				position: absolute;
+		        top: 0;
+		        left: 20px;
+				display: block;
+		        height: 48px;
+		        width: 48px;
+
+		        .c-search.m-small & {
+
+					left: 12px;
+		        	height: 24px;
+		        	width: 24px;
+		        }
+			}
+
+			&.m-destinations {
+
+				&:before {
+
+					background: url('#{$path-images}/icon-offer.svg');
```

Comment:
always being advocating keeping SVGs in Blade side. Not sure about these. I'd like concistency but...